controllers/platform/services/services.go

```diff
@@ -265,6 +269,9 @@ func (j JobServiceHandler) GetContainerName() string {
 }
 
 func (j JobServiceHandler) GetServiceImageName(persistenceName string) string {
+	if len(cfg.GetCfg().JobsServiceImageTag) > 0 {
+		return cfg.GetCfg().JobsServiceImageTag
```

Comment:
Not sure if this is good, or at least it's not aligned to what we had. Let me explain:
In order to get the proper jobs-service image, right now, two variables are considered, the "persitenceName"  and the "tag" to use.
For example, if no persistence is configured, "ephemeral" persistence will be considered, and thus, the returned image will be  kogito-jobs-service-ephemeral-nightly:latest.
On the other hand, if postgresql is configured, then, kogito-jobs-service-postgresql-nightly:latest will be used instead.

So, the image might vary depending on the persistence type we are using.

With your change, only one image JobsServiceImageTag is admitted.  Is that correct?

Reply:
Oh yeah, I forgot about that, I'll add two more configuration options there to solve this.
Thank you!